bones/codeBone.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please remove this line, as it is not required anymore (see [here](https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3))